backend/src/oe/service/op-block-header.service.ts

```diff
@@ -24,7 +25,14 @@ export class OpBlockHeaderService {
         mediantime,
       } = await bitcoinApi.$getBlock(blockHash);
 
-      const { totalfee, subsidy } = await bitcoinApi.$getBlockStats(blockHash);
+      let fees: number = 0;
+      let subsidy: number = DEFAULT_SUBSIDY;
+
+      if (blockHeight.value > 0) {
```

Comment:
Due to an issue in the blockchain's API, it does not return any stat value for the zeroth block, instead throws an error. To avoid the error we are first checking the block height